boards/avr-rss2/Makefile.include

```diff
@@ -0,0 +1,23 @@
+# configure the terminal program
+PORT_LINUX  ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+# refine serial port information for pyterm
+BAUD        ?= 115200
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# PROGRAMMER defaults to wiring which is the internal flasher via USB
+# using avrdude. Can be overridden for debugging (which requires changes
+# that require to use an ISP)
+PROGRAMMER ?= stk500v2
+# 
+# see /usr/include/asm-generic/termbits.h for availabel baudrates on your linux system
+#FFLAGS_EXTRA += -b 0010005
+# avoid error if mcu signature doesn't match
+#FFLAGS_EXTRA += -F
+
```

Comment:
What would those `FFLAGS_EXTRA` be used for?
If they are not needed, just remove them.